<launch>
  <arg name="LAUNCH_DYNAMIC_TF_PUBLISHER" default="true" />
  <arg name="models" default="$(find jsk_interactive_marker)/launch/models/urdf_model_markers.yaml" />
  <node pkg="jsk_interactive_marker" type="urdf_model_marker" name="jsk_model_marker_interface" clear_params="true" output="log" >
    <rosparam command="load" param="model_config" file="$(arg models)"/>
    <param name="use_dynamic_tf" value="$(arg LAUNCH_DYNAMIC_TF_PUBLISHER)"/>
  </node>

  <group if="$(arg LAUNCH_DYNAMIC_TF_PUBLISHER)">
    <node pkg="dynamic_tf_publisher" type="tf_publish.py" name="dynamic_tf_publisher_for_urdf_marker">
      <param name="use_cache" type="bool" value="false"/>
      <param name="check_update" type="bool" value="true"/>
    </node>

  </group>
</launch>